### PR TITLE
[JENKINS-55127] make parallel input stages individually selectable

### PR DESCRIPTION
**Pull request by NicuPascu**
parallel seq stages don't have the isParallel flag set so we need to
check the "firstParent" vs "parent" properties

# Description

See [JENKINS-55127](https://issues.jenkins-ci.org/browse/JENKINS-55127).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by NicuPascu** [comment]
seq_inputs

File: blueocean-dashboard/src/main/js/components/karaoke/components/Pipeline.jsx [hunk 1]
```diff
@@ -270,8 +270,9 @@ export default class Pipeline extends Component {
                 pathArray.shift();
                 nextPath = `/${pathArray.join('/')}`;
             }
-            // check whether we have a parallel node
-            const isParallel = nextNode.isParallel;
+            // check whether we have a parallel node or seq parallel nodes
+            const isParallel = nextNode.isParallel || (nextNode.firstParent && nextNode.parent != nextNode.firstParent);
```
[the remark this answers is not in the document]
Comment (NicuPascu) [reply blueocean-dashboard/src/main/js/components/karaoke/components/Pipeline.jsx]: As per our discussion, this needs to be like this so it returns undefined in some scenarios (independent nodes)

[the remark this answers is not in the document]
Comment (davidbees) [reply blueocean-dashboard/src/main/js/components/karaoke/components/Pipeline.jsx]: 👍 Thanks for the clarification.

**Comment by NicuPascu** [comment]
greenest_states_cover

**Pull request merged**